### PR TITLE
blog post shiny rock

### PR DESCRIPTION
**Pull request by MartinRst**

### CODE REVIEW

File: content/blogs/2023-08-22-generating-fake-data.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+        namespace: shiny_rocks
+        flowId: dbt_run
+      - type: io.kestra.core.models.conditions.types.ExecutionStatusCondition
+        in:
+          - SUCCESS
+```
+
+## Conclusion ##
+
+For those who wish to dive deeper, we have made the entire Shiny Rocks dataset available for your exploration and usage.
```
Comment (Ben8t) [inline content/blogs/2023-08-22-generating-fake-data.md:416]: Add link to https://github.com/kestra-io/shiny_rocks

File: content/blogs/2023-08-22-generating-fake-data.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+title: "Generating Fake Data to Create End-to End Orchestration Projects"
+description: " generates dynamic fake data for end-to-end orchestration projects with Kestra, for realistic data use cases and hands-on experience"
+date: 2023-08-22T16:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "Bpimpaud"
+image: /blogs/2023-08-22-generating-fake-data.jpg
+---
+Welcome to the world of Shiny Rocks! Although this company might not exist outside Kestra it has its own purpose.
```
Comment (Ben8t) [inline content/blogs/2023-08-22-generating-fake-data.md:11]: Add link to https://github.com/kestra-io/shiny_rocks on the "Shiny Rocks"

File: content/blogs/2023-08-22-generating-fake-data.md [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+- Services: Illustrates the services users access on their smartphones - maps, contacts, music, etc.
+
+By using distribution functions, we've given these datasets diverse and random attributes.
+
+For example we want two products, one being present in 64% of orders and the other in 36% of them. We do the same with marketing UTM sources, payment methods, product service usages, etc...
+
+```python
+class ShinyRockProvier(BaseProvider):
+
+    def order(self, order_date) -> str:
```
Comment (Ben8t) [inline content/blogs/2023-08-22-generating-fake-data.md:41]: remove -> str

File: content/blogs/2023-08-22-generating-fake-data.md [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+labels:
+  tag: analytics
+
+tasks:
+  - id: workingdir
+    type: io.kestra.core.tasks.flows.WorkingDirectory
+    tasks:
+      - id: cloneRepository
+        type: io.kestra.plugin.git.Clone
+        url: https://github.com/Ben8t/shiny_rocks
```
Comment (Ben8t) [inline content/blogs/2023-08-22-generating-fake-data.md:255]: update link https://github.com/kestra-io/shiny_rocks

File: content/blogs/2023-08-22-generating-fake-data.md [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+  tag: producer
+
+tasks:
+  - id: working_dir
+    type: io.kestra.core.tasks.flows.WorkingDirectory
+    tasks:
+
+      - id: clone
+        type: io.kestra.plugin.git.Clone
+        url: https://github.com/Ben8t/shiny_rocks
```
Comment (Ben8t) [inline content/blogs/2023-08-22-generating-fake-data.md:96]: update link https://github.com/kestra-io/shiny_rocks

File: content/blogs/2023-08-22-generating-fake-data.md [hunk 6]
```diff
@@ -0,0 +1,421 @@
...
+
+We often face the challenge of presenting concepts without real data, due to privacy concerns or practicalities.
+
+Shiny Rocks is a fictional company creating smartphones. "Those shiny rocks in our pockets". It provides a means to showcase various data use cases in a realistic manner. In this post, we are not just introducing this mock company, but also offering datasets it "produces" for your hands-on experience. Plus, in the future, anticipate more of Shiny Rocks; it's going to be our gimmick to bring plausible data scenarios, especially when we dive deep into use cases with Kestra.
+
+![architecture](/blogs/2023-08-22-generating-fake-data/architecture.png)
+
+## Generating fake data ##
+
+While we admire dbt [jaffle-shop's portrayal](https://github.com/dbt-labs/jaffle_shop) of a fictional e-commerce store, we wanted something more dynamic, especially when showcasing orchestration patterns. Thus, Shiny Rocks was born. Unlike static files that don't change, Shiny Rocks comes with custom scripts that generate fresh data every day. These scripts rely on a mix of mathematical functions to simulate various trends for our imaginary company.
```
Comment (anna-geller) [inline content/blogs/2023-08-22-generating-fake-data.md:21]:
```suggestion
Unlike static files that don't change, Shiny Rocks comes with custom scripts that generate fresh data every day. These scripts rely on a mix of mathematical functions to simulate various trends for our imaginary company.
```

cut it, there is no need to mention jaffle shop here, not relevant in this context

Comment (Ben8t) [reply content/blogs/2023-08-22-generating-fake-data.md:21]: alright 👍

**Pull request merged**